.github/workflows/nightly-test.yml

```diff
@@ -18,15 +18,13 @@ env:
   GCLOUD_SERVICE_KEY_JSON_BASE64: ${{ secrets.GCLOUD_SERVICE_KEY_JSON_BASE64 }}
   GOOGLE_COMPUTE_REGION: ${{ vars.GOOGLE_COMPUTE_REGION }}
   GOOGLE_PROJECT_ID: ${{ vars.GOOGLE_PROJECT_ID }}
-  HC_CONSUL_LICENSE: ${{ secrets.HC_CONSUL_LICENSE }}
-  HC_VAULT_LICENSE: ${{ secrets.HC_VAULT_LICENSE }}
   INSTRUQT_TOKEN: ${{ secrets.INSTRUQT_TOKEN }}
 jobs:
   # This should be changed long term: we build a new container
   # every night even though the majority of the time nothing changes.
   # Figure out a way to trigger only when the source changes, or
   # the configuration changes
-  license-cloud-client:
+  update-cloud-client:
```

Comment:
We're no longer updating the license, so change the step name to something more applicable.